src/Constants.js

```diff
@@ -1,51 +1,51 @@
  * @property {Object} [units]
- * @property {string} [units.seconds='seconds'] The name of the 'seconds' units
- * @property {string} [units.minutes='minutes'] The name of the 'minutes' units
- * @property {string} [units.hours='hours'] The name of the 'hours' units
- * @property {string} [units.days='days'] The name of the 'days' units
+ * @property {string} [units.seconds='seconds'] The name of the "seconds" unit
+ * @property {string} [units.minutes='minutes'] The name of the "minutes" unit
+ * @property {string} [units.hours='hours'] The name of the "hours" unit
+ * @property {string} [units.days='days'] The name of the "days" unit
  * @property {Boolean} [units.pluralS='false'] Whether to force removing the "S" which marks the plural when the value is lower than 2
  */
 exports.GiveawayMessages = {};
 
 /**
- * The start options for new giveaways
+ * The start options for new giveaways.
  * @typedef GiveawayStartOptions
  *
- * @property {number} time The giveaway duration
- * @property {number} winnerCount The number of winners for the giveaway
- * @property {string} prize The giveaway prize
- * @property {Discord.User} [hostedBy] The user who hosts the giveaway
- * @property {Boolean} [botsCanWin] Whether the bots are able to win a giveaway.
+ * @property {number} time The giveaway duration.
+ * @property {number} winnerCount The number of winners for the giveaway.
+ * @property {string} prize The giveaway prize.
+ * @property {Discord.User} [hostedBy] The user who hosts the giveaway.
+ * @property {Boolean} [botsCanWin] If bots can win the giveaway.
  * @property {Discord.PermissionResolvable[]} [exemptPermissions] Members with any of these permissions won't be able to win a giveaway.
  * @property {Function} [exemptMembers] Function to filter members. If true is returned, the member won't be able to win the giveaway.
  * @property {BonusEntry[]} [bonusEntries] An array of BonusEntry objects.
- * @property {Discord.ColorResolvable} [embedColor] The giveaway embeds color when they are running
- * @property {Discord.ColorResolvable} [embedColorEnd] The giveaway embeds color when they are ended
- * @property {Discord.EmojiIdentifierResolvable} [reaction] The reaction to participate to the giveaways
- * @property {GiveawayMessages} [messages] The giveaway messages
- * @property {any} [extraData] The extra data value for this giveaway
- * @property {LastChanceOptions} [lastChance] The last chance system options
+ * @property {Discord.ColorResolvable} [embedColor] The color of the giveaway embed when it is running.
+ * @property {Discord.ColorResolvable} [embedColorEnd] The color of the giveaway embed when it has ended.
+ * @property {Discord.EmojiIdentifierResolvable} [reaction] The reaction to participate in the giveaway.
+ * @property {GiveawayMessages} [messages] The giveaway messages.
+ * @property {any} [extraData] The extra data value(s) for this giveaway.
```

Comment:
I mean it can only have 1 value = singular. Setting it as an obj or an array does not mean that the property has multiple values. Or how do you interpret that?

Reply:
Since this is the only place the user can add their extra data values, I think that `value(s)` would be better.

Reply:
but extraData can only have 1 value
Only the value of extraData can then have multiple values/properties if extraData is an array or obj
so "values/properties of value of extraData !== values of extraData" in my opinion

Reply:
Maybe we can... just remove the value x)
This would be the same: `The extra data for this giveaway (to store any additional things such as server requirements).`

Reply:
and thank you for the PR, great ideas 👍

Reply:
> Maybe we can... just remove the value x)
> This would be the same: `The extra data for this giveaway (to store any additional things such as server requirements).`

Sounds great.